app/views/SubmissionsView.scala.html

```diff
@@ -89,6 +100,61 @@ <h2 class="govuk-heading-s">
           }
         </tbody>
       </table>
+
+      <nav class="govuk-pagination" role="navigation" aria-label="results">
+
+        @if(offset > 0) {
+          <div class="govuk-pagination__prev">
+            <a class="govuk-link govuk-pagination__link" href="@routes.SubmissionsController.onPageLoad(service, status, created, Some(offset - limit))" rel="prev">
+              <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+                <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+              </svg>
+              <span class="govuk-pagination__link-title">@messages("pagination.previous")</span></a>
+          </div>
+        }
+        <ul class="govuk-pagination__list">
+
+          @if(currentPage > 2) {
+            <li class="govuk-pagination__item">
+              <a class="govuk-link govuk-pagination__link" href="@routes.SubmissionsController.onPageLoad(service, status, created, None)" aria-label="Page 1">
+                1
+              </a>
+            </li>
+          }
+          @if(currentPage > 3) {
+            <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
+          }
+
+          @for(i <- (currentPage - 1).max(1) to (currentPage + 1).min(totalCount / limit)) {
+            <li class="govuk-pagination__item @if(i == currentPage) {govuk-pagination__item--current}">
+              <a class="govuk-link govuk-pagination__link" href="@routes.SubmissionsController.onPageLoad(service, status, created, Some((i - 1) * limit))" aria-label="Page 7" aria-current="page">
```

Comment:
Should the page number in the aria-label here be dynamic?

Reply:
[Sorted with latest commit :)](https://github.com/hmrc/dms-submission-admin-frontend/pull/7/commits/16cc18066af12280b59eee5cfdbcf3426bc31522)